home/templates/home/index.html

```diff
@@ -3,10 +3,88 @@
+                    suscipit faucibus urna.
+                    </p>
+                </div>
+                <h3>Hackathon 04 Title</h3>
+                <div>
+                    <p>
+                    Nam enim risus, molestie et, porta ac, aliquam ac, risus. Quisque lobortis.
+                    Phasellus pellentesque purus in massa. Aenean in pede. Phasellus ac libero
+                    ac tellus pellentesque semper. Sed ac felis. Sed commodo, magna quis
+                    lacinia ornare, quam ante aliquam nisi, eu iaculis leo purus venenatis dui.
+                    </p>
+                    <ul>
+                    <li>List item one</li>
+                    <li>List item two</li>
+                    <li>List item three</li>
+                    </ul>
+                </div>
+                <h3>Hackathon 05 Title</h3>
+                <div>
+                    <p>
+                    Cras dictum. Pellentesque habitant morbi tristique senectus et netus
+                    et malesuada fames ac turpis egestas. Vestibulum ante ipsum primis in
+                    faucibus orci luctus et ultrices posuere cubilia Curae; Aenean lacinia
+                    mauris vel est.
+                    </p>
+                    <p>
+                    Suspendisse eu nisl. Nullam ut libero. Integer dignissim consequat lectus.
+                    Class aptent taciti sociosqu ad litora torquent per conubia nostra, per
+                    inceptos himenaeos.
+                    </p>
+                </div>
+            </div>
+        </div><!---col div-->
+    <!--carousel-->  
+        <div class = "col-6 col-sm-12  col-md-6">
+            <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+                <div class="carousel-inner">
+                    <div class="carousel-item active">
+                    <img src="" class="d-block w-50" alt="...">
+                    </div>
+                    <div class="carousel-item">
+                    <img src="..." class="d-block w-50" alt="...">
+                    </div>
+                    <div class="carousel-item">
+                    <img src="..." class="d-block w-50" alt="...">
+                    </div>
+                </div>
+                <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
+                    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                    <span class="sr-only">Previous</span>
+                </a>
+                <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next">
+                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                    <span class="sr-only">Next</span>
+                </a>
+            </div>
+
+        </div><!--col div-->
+
+        
+    </div><!--row div-->
+</div><!--container div-->

```

Comment:
We don't need the code-comments for everything. Can you remove these? (and all of the ones above from this file)